src/ClassicUO.Client/LegionScripting/API.py

```
@@ -0,0 +1,286 @@
+    pass
+
+def GetAllMobiles():
+    pass
+
+def GetTile(x: int, y: int):
+    pass
+
+def CreateGump(acceptMouseInput: bool = True, canMove: bool = True):
+    pass
+
+def AddGump(g: Any):
```

Ruff (0.8.2)

Undefined name `Any`

(F821)